libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCredentials.h

```diff
@@ -25,17 +25,6 @@
 #import <Foundation/Foundation.h>
 
 /**
- @enum Logging levels to control the verbosity of log output based on the severity of the event being logged.
- */
-typedef NS_ENUM(NSUInteger, SFOAuthLogLevel) {
```

Comment:
The default is `Debug` in debug builds and `Error` in prod builds. Are you suggesting `OAuth` should continue to have its own component (this breaks the design we seek to achieve with the new framework within the SDK)?